src/sdk.ts

```diff
@@ -22,27 +21,35 @@ import {
 } from './utils';
 import { ERC20Token } from './ERC20Token';
 import { ZERO_ADDRESS } from './constants';
-import { RelayingServicesConfiguration, SmartWallet } from './interfaces';
+import {
+    RelayingServicesAddresses,
+    RelayingServicesConfiguration,
+    SmartWallet
+} from './interfaces';
 import { Contracts } from './contracts';
 
 export class DefaultRelayingServices implements RelayingServices {
-    private readonly web3Instance: Web3;
-    private readonly account?: Account;
-    private developmentAccounts: string[];
-    private relayProvider: RelayProvider;
-    private contracts: Contracts;
+    protected readonly web3Instance: Web3;
+    protected readonly account?: Account;
+    protected developmentAccounts: string[];
+    protected relayProvider: RelayProvider;
+    protected contracts: Contracts;
```

Comment:
Why do we need to set those fields as `protected`? Is this class intended to be extended? Furthermore, if we don't plan to change them, I would set them as final, if possible.

Reply:
This code was added by @JONAF2103, This code is for implement and  extend by the user.